x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/waterfall_chart.tsx

```diff
@@ -219,6 +228,7 @@ export const WaterfallChart = ({
           </EuiFlexItem>
         </EuiFlexGroup>
         {shouldRenderLegend && <Legend items={legendItems!} render={renderLegendItem!} />}
+        <WaterfallFlyout />
```

Comment:
I should maybe defer the render to the Waterfall to the WaterfallChartWrapper like other renders.